index.html

```diff
@@ -1,13 +1,259 @@
 <!DOCTYPE html>
 <html lang="en">
   <head>
-    <meta charset="utf-8" />
+    <meta
+      charset="utf-8"
+      name="viewport"
+      content="width=device-width, initial-scale=1"
+    />
 
     <link href="styles.css" rel="stylesheet" />
 
-    <title>Rocket Academy</title>
+    <title>James' Portfolio</title>
+    <link rel="icon" type="image/x-icon" href="./images/favicon.ico" />
   </head>
-  <body>
-    <h1>Hello, world!</h1>
+  <body id="top">
+    <header>
+      <div id="top-navbar">
+        <a href="#top" id="logo">
+          <text>queky<span class="font-highlight">portfolio</span></text>
+        </a>
+        <nav id="top-nav-items">
+          <a href="#about" class="nav-link active"
+            ><span class="font-highlight">01.</span> About</a
+          >
+          <a href="#projects" class="nav-link"
+            ><span class="font-highlight">02.</span> Projects</a
+          >
+          <a href="#contact" class="nav-link"
+            ><span class="font-highlight">03.</span> Contact</a
+          >
+        </nav>
+      </div>
+    </header>
+
+    <main>
+      <section id="about">
+        <div class="container">
+          <div class="text-div">
+            <h1>
+              Hey there,<br />I am
+              <span class="font-highlight scale">James</span>,<br />
+              Bootcamp Student
+            </h1>
+
+            <p>
+              Currently residing in Singapore. I love building
+              <span class="font-highlight">interactive</span>, digital
+              experiences on the web.
+            </p>
+
+            <button id="about-cta-btn">Let's get in touch!</button>
+          </div>
+
+          <figure class="key-visuals">
+            <img
+              class="animate"
+              src="images/qp-hero-image.png"
+              alt="qp-hero-image"
+            />
+          </figure>
+        </div>
+      </section>
+
+      <section id="scroll-arrow-section">
+        <div class="scroll-arrow-prompt">
+          <text>scroll down</text>
+          <img
+            class="animate"
+            id="arrow-prompt"
+            src="images/qp-arrow-prompt.png"
+          />
+        </div>
+      </section>
+
+      <section id="about-2">
```

Comment:
no harm naming this section with an id like `skills-and-tools`. makes it easier for other people to understand which section this is without referencing the actual page